src/encryption.ts

```diff
@@ -1,70 +1,74 @@
 interface GeneralRecipient {
-  encrypted_key?: string;
-  header?: Record<string, any>;
+  encrypted_key?: string
+  header?: Record<string, any>
```

Comment:
Nice, I didn't like the `;` here. For some reason I thought they where needed :)